bitcoin/Cargo.toml

```diff
@@ -32,7 +32,7 @@ std = ["secp256k1/std", "bitcoin_hashes/std", "bech32/std", "bitcoin-internals/s
 no-std = ["hashbrown", "core2/alloc", "bitcoin_hashes/alloc", "secp256k1/alloc"]
 
 [package.metadata.docs.rs]
-features = [ "std", "secp-recovery", "base64", "rand", "serde", "bitcoinconsensus" ]
+features = [ "std", "base64", "rand-sd", "rand", "serde", "bitcoinconsensus", "secp-recovery", "secp-lowmemory", "bitocinconsensus-std", "rust_v_1_53" ]
```

Comment:
In fc5d7bb39c5d58d7d8261a0f5d5672c8a4bcb89d:

I guess `rand-sd` should be `rand-std`?